fe/fe-core/src/main/java/com/starrocks/transaction/OlapTableTxnLogApplier.java

```diff
@@ -127,9 +127,9 @@ public void applyVisibleLog(TransactionState txnState, TableCommitInfo commitInf
                         if (!txnState.tabletCommitInfosContainsReplica(tablet.getId(), replica.getBackendId(),
                                 replica.getState())
                                 || errorReplicaIds.contains(replica.getId())) {
```

Comment:
```suggestion
                        if (transactionState.checkReplicaNeedSkip(tablet, replica, partitionCommitInfo)
                                || errorReplicaIds.contains(replica.getId())) {
```

Reply:
better to not set `lastFailedVersion` on the cloned healthy replica

Reply:
tks review